features/cubic-bezier-easing.yml

```diff
@@ -0,0 +1,6 @@
+name: cubic-bezier() easing
+description: The `cubic-bezier()` easing function interpolates along a smooth curve, creating animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.
+spec: https://drafts.csswg.org/css-easing-2/#cubic-bezier-easing-functions
+group: css
+compat_features:
+  - css.types.easing-function.cubic-bezier
```

Comment:
I sent https://github.com/mdn/browser-compat-data/issues/24682 due to all the keyword values not being in BCD.